veocean-data-farming/df-background.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Intuition and background on the Data Farming program.
+---
+# Data Farming Background
+
+It rewards OCEAN to liquidity providers (stakers) as a function of consume volume and liquidity. It’s like DeFi liquidity mining, but tuned for data consumption. DF’s aim is to achieve a minimum supply of data for network effects to kick in, and once the network flywheel is spinning, to increase growth rate.
+
+
+## Active Work to Drive APY
+
+Data Farming is not a wholly passive activity. The name of the game is to drive Data Consume Volume (DCV). High APYs happen only when there is sufficiently high DCV. High DCV means publishing and consuming truly useful datasets (or algorithms).
+
+Thus, if you really want to max out your APY:
+- create & publish datasets (and make $ in selling them) — or work with people who can
+- consume the datasets (to make $) — or work with people who can
+- go stake on them, and finally claim the rewards.
+
+Driving DCV for publishing & consuming is your challenge. It will take real work. And then the reward is APY. It’s incentives all the way down:)
+
+## Reward Function
+
+The Reward Function (RF) governs how active rewards are allocated to stakers.
+
+Rewards are calculated as follows:
+1. Distribute OCEAN across each asset based on rank: highest-DCV asset gets most OCEAN, etc.
+1. For each asset and each veOCEAN holder:
```

Comment:
```suggestion
- Distribute OCEAN across each asset based on rank: highest-DCV asset gets most OCEAN, etc.
- For each asset and each veOCEAN holder:
```

Reply:
Your feedback is incorrect.
The correct way of creating a numbered list in markdown is.
```markdown
1. One
1. Two
1. Three
```